fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/api/registry/VillagerPlantableRegistry.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.registry;
+
+import java.util.Set;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemConvertible;
+
+import net.fabricmc.fabric.impl.content.registry.VillagerPlantableRegistryImpl;
+
+/**
+ * Registry of items that farmer villagers can plant on farmland.
+ */
+public interface VillagerPlantableRegistry {
+	VillagerPlantableRegistry INSTANCE = new VillagerPlantableRegistryImpl();
```

Comment:
We don't use the `INSTANCE` pattern anymore. Could you turn the methods here into static methods?

Reply:
Done!